documentation/docs/protocol_specification/components/congestion_control.md

```diff
@@ -1,87 +1,186 @@
 description: Every network has to deal with its intrinsic limited resources. GoShimmer uses congestion control algorithm to regulate the influx of messages in the network with the goal of maximizing throughput (messages/bytes per second) and minimizing delays.
-image: /img/protocol_specification/congestion_control_algorithm_infographic.png
+image: /img/protocol_specification/congestion_control_algorithm_infographic_new.png
 keywords:
+
 - node
 - congestion control algorithm
 - honest node
 - message
 - access mana
 - malicious nde
 - scheduling
+
 ---
 
 # Congestion Control
 
-Every network has to deal with its intrinsic limited resources in terms of bandwidth and node capabilities (CPU and storage). In this document, we present a congestion control algorithm to regulate the influx of messages in the network with the goal of maximizing throughput (messages/bytes per second) and minimizing delays. Furthermore, the following requirements must be satisfied:
+Every network has to deal with its intrinsic limited resources in terms of bandwidth and node capabilities (CPU and
+storage). In this document, we present a congestion control algorithm to regulate the influx of messages in the
+network with the goal of maximizing throughput (messages/bytes per second) and minimizing delays. Furthermore, the
+following requirements must be satisfied:
 
-*   _Consistency_. If a message is written by one honest node, it shall be written by all honest nodes within some delay bound.
-*   _Fairness_. Nodes can obtain a share of the available throughput depending on their access Mana. Throughput is shared in such a way that an attempt to increase the allocation of any node necessarily results in the decrease in the allocation of some other node with an equal or smaller allocation (max-min fairness).
-*   _Security_. Malicious nodes shall be unable to interfere with either of the above requirements.
+* _Consistency_. If a message is written by one honest node, it shall be written by all honest nodes within some
+  delay bound.
+* _Fairness_. Nodes can obtain a share of the available throughput depending on their access Mana. Throughput is
+  shared in such a way that an attempt to increase the allocation of any node necessarily results in the decrease
+  in the allocation of some other node with an equal or smaller allocation (max-min fairness).
+* _Security_. Malicious nodes shall be unable to interfere with either of the above requirements.
 
-[![Congestion Control](/img/protocol_specification/congestion_control_algorithm_infographic.png)](/img/protocol_specification/congestion_control_algorithm_infographic.png)
+[![Congestion Control](/img/protocol_specification/congestion_control_algorithm_infographic_new.png)](/img/protocol_specification/congestion_control_algorithm_infographic_new.png)
 
 Further information can be found in the paper [Access Control for Distributed Ledgers in the Internet of Things: A Networking Approach](https://arxiv.org/abs/2005.07778).
```

Comment:
You can also add this link which includes attack analysis https://arxiv.org/abs/2107.10238